inference/inference.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import concurrent.futures
+from typing import Tuple
+import time
+from tqdm import tqdm
+from concurrent.futures import wait
+import time
+
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from tensorflow.keras.models import load_model
+
+import autokeras as ak
+
+
+def train_model(x_train: np.ndarray, y_train: np.ndarray) -> ak.TextRegressor:
+    reg = ak.TextRegressor(overwrite=True, max_trials=1)
+    # Feed the text regressor with training data.
+    reg.fit(x_train, y_train, epochs=2)
+    # Predict with the best model.
+
+    model = reg.export_model()
+
+    try:
+        model.save("model_autokeras", save_format="tf")
+    except Exception:
+        model.save("model_autokeras.h5")
+
+    return reg
+
+
+def get_data(path: str) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    train_dataset = pd.read_csv(path,
+                                na_values="?", comment='\t',
+                                sep=",", skipinitialspace=True)
+
+    X = train_dataset.excerpt
+    y = train_dataset.target
+
+    train_X, val_X, train_y, val_y = train_test_split(X, y, random_state=1)
+
+    return np.array(train_X), np.array(train_y), np.array(val_X)
+
+
+def predict(model: ak.TextRegressor, x: np.ndarray) -> np.ndarray:
+    # replace with real model
+    # time.sleep(0.001)
+    return model.predict(x)
+
+
+def run_inference(x_test: np.ndarray, batch_size: int = 64) -> np.ndarray:
+    model = load_model("model_autokeras", custom_objects=ak.CUSTOM_OBJECTS)
+
+    y_pred = []
+    for k in range(0, 100):
```

Comment:
This is just to have enough load on the CPU so that parallelization has at least some benefits